### PR TITLE
Bump gulp to >4

### PR DESCRIPTION
**Pull request by xlfe**
As requested, a PR to bump gulp to 4

Reference 
https://github.com/GoogleCloudPlatform/stackdriver-errors-js/pull/92#issuecomment-852246993

### CODE REVIEW

**Comment by xlfe** [comment]
@googlebot I signed it!

**Comment by jmasukawa** [comment]
@steren FYI

File: package.json [hunk 1]
```diff
@@ -31,25 +31,25 @@
...
-    "nise": "^1.4.10",
-    "rimraf": "^2.6.3",
+    "nise": "^1.5.3",
+    "rimraf": "^2.7.1",
     "vinyl-buffer": "^1.0.1",
     "vinyl-source-stream": "^2.0.0"
   },
   "dependencies": {
-    "stacktrace-js": "^2.0.0"
+    "core-js": "^3.11.3",
```
Comment (steren) [inline package.json:52]: `core-js` has been moved to devDependencies in https://github.com/GoogleCloudPlatform/stackdriver-errors-js/commit/28f6307799c59b7960d1c7a374b67b173aecb041#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519, can you do the same?

**Review by steren — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (xlfe) [reply package.json]: @steren ah sorry fixed below

**Pull request merged**